README.md

```diff
@@ -123,6 +123,9 @@ Use the following environment variables to configure the application:
 | <tt>GITHUB_ACCESS_TOKEN</tt>                  |                               | GitHub access token. Instruction for creating a token can be found [here](./docs/gh-token.md). If not provided, the owners validating functionality may not work properly. For example, you may reach the API calls quota or, if you are setting GitHub Enterprise base URL, an unauthorized error may occur.                                                                                                                                                   |
 | <tt>GITHUB_BASE_URL</tt>                      | `https://api.github.com/`     | GitHub base URL for API requests. Defaults to the public GitHub API but can be set to a domain endpoint to use with GitHub Enterprise.                                                                                                                                                                                                                                                                                                                          |
 | <tt>GITHUB_UPLOAD_URL</tt>                    | `https://uploads.github.com/` | GitHub upload URL for uploading files. <br> <br>It is taken into account only when `GITHUB_BASE_URL` is also set. If only `GITHUB_BASE_URL` is provided, this parameter defaults to the `GITHUB_BASE_URL` value.                                                                                                                                                                                                                                                |
+| <tt>GITHUB_APP_ID</tt>                        |                               | Github App ID for authentication. This replaces the `GITHUB_ACCESS_TOKEN`.                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Could you also create (short or even raw note) on how to use it with it? Similar to the token access doc: https://github.com/mszostok/codeowners-validator/blob/main/docs/gh-token.md

---
How I tested that:
1. Created a GitHub App (https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app).
2. Added the read-only access to members in org:
    ![Screen Shot 2022-04-14 at 15 03 45](https://user-images.githubusercontent.com/17568639/163396399-a55cb65c-5d05-4a47-bfa3-487fa97869f5.png)

4. Installed the App in my org.

**Obtaining the installation ID:**
1. With a given JS snippet https://github.com/octokit/auth-app.js/#authenticate-as-github-app-json-web-token, I generated a JWT.
2. With the JWT, I listed installations:
    ```bash
    curl -i -X GET \
    -H "Authorization: Bearer <token>" \
    -H "Accept: application/vnd.github.v3+json" \
    https://api.github.com/app/installations
    ```

So finally I had the `GITHUB_APP_ID` and `GITHUB_APP_PRIVATE_KEY ` from GitHub APP setting, and the `GITHUB_APP_INSTALLATION_ID` based on the executed curl. 

Is there an easier way?

Reply:
There is an easier way that doesn't require that JWT stuff (which is quite complex to do if you don't do it often IMO). I've documented it, let me know what you think!

Reply:
Looks great! Thanks!